### DISCUSSION TITLE
Bounce Response forward

### DISCUSSION THREAD

**Discussion by arunkuttytry** [postalserver/postal#2225] [Help with using Postal]
Hi Sir,

I got lot of bounce, i am using gmass, can you please tell me how to send bounce email forward to sender email ,

Please do help me 

![image](https://user-images.githubusercontent.com/107031529/200068376-34bc5e55-b5d8-40b9-9515-11200d7eaedc.png)

**Comment by willpower232** [comment]
You can forward all bounces to a single email address via the routing https://docs.postalserver.io/other/auto-responders-and-bounces

but if you want to have them be forwarded separately then I'm not sure how you can do that.
